_data/README.md

```diff
@@ -63,6 +63,16 @@ Use a [validator](https://www.yamllint.com/) to make sure the Yaml is valid.
   twitter: ""
   githubUrl: ""
   blogPost: ""
+
+# Notice the green cirlce
+- Green circle: build is busy
+- Green V : build succeeded, new yaml is visible on the website
+- Red X : build failed, probably invalid Yaml
+
+# Decide whether the cover image of video is to be used
+- If the video is to be used as cover image, then it's OK as it
+- If there is no video or the video is not to be used as cover image, remove the best picture from the gallery and use that as image
+
```

Comment:
The code part is what the actual Yaml looks like, this is some more general information, probably to be added at the bottom of the README instead

Meaning: in the events.yml, there is no key "Green circle" etc

Reply:
I moved it already to the bottom part